vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -850,6 +862,42 @@ def __init__(
 
         self.top_k = top_k
 
+        self.num_fused_shared_experts = \
```

Comment:
Since this feature is only supported by `rocm_aiter_moe` can you add an assert that `num_fused_shared_experts` is 0 when rocm is not enabled?